src/components/ContributionsEpicReminder.tsx

```diff
@@ -0,0 +1,241 @@
+    display: none;
+
+    + [data-target='toggle'] [data-target='close'] {
+        display: none;
+    }
+
+    :checked ~ [data-target='pane'] {
+        display: block;
+    }
+
+    :checked + [data-target='toggle'] [data-target='close'] {
+        display: block;
+    }
+
+    :checked + [data-target='toggle'] [data-target='open'] {
+        display: none;
+    }
+`;
+
+const toggleStyles = css`
+    display: block;
+    position: relative;
+    text-align: right;
+    padding-right: ${space[4]}px;
+`;
+
+const openButtonStyles = css`
+    ${textSans.medium({
+        fontWeight: 'bold',
+    })};
+    text-decoration: underline;
+    cursor: pointer;
+`;
+
+const closeButtonStyles = css`
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    position: absolute;
+    bottom: -47px;
+    right: 0;
+`;
+
+const paneStyles = css`
+    display: none;
+`;
+
+const lineWrapperStyles = css`
+    margin: ${space[2]}px auto;
+`;
+
+const containerStyles = css`
+    padding: 0 ${space[1]}px;
+`;
+
+const remindHeading = css`
+    ${headline.xxsmall({ fontWeight: 'bold' })};
+    margin: 0 ${space[5]}px ${space[2]}px 0;
+`;
+
+const successTextStyles = css`
+    margin: 0 auto ${space[2]}px;
+    ${body.medium()};
+`;
+
+const linkStyles = css`
+    color: ${palette.neutral[7]};
+`;
+
+const formWrapper = css`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    ${from.tablet} {
+        flex-direction: row;
+        align-items: flex-end;
+        justify-content: flex-start;
+    }
+`;
+
+const inputWrapper = css`
+    width: 100%;
+    margin-bottom: ${space[2]}px;
+    flex-grow: 1;
+
+    ${from.tablet} {
+        width: auto;
+        margin-right: ${space[2]}px;
+        margin-bottom: 0;
+    }
+`;
+
+const formTextStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
+    font-style: italic;
+    margin-top: ${space[1]}px;
+`;
+
+const errorTextStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
+    color: ${palette.error.main};
+    font-style: italic;
+    margin-top: ${space[1]}px;
+`;
+
+export const ContributionsEpicReminder: React.FC<ReminderFields> = ({
+    reminderDate,
+    reminderDateAsString,
+}: ReminderFields) => {
+    const unique = new Date().valueOf();
+    return (
+        <div data-target={`contributions-epic-reminder`} className={rootStyles}>
+            <input id={`epicSwitch${unique}`} type="checkbox" className={checkboxStyles} />
+            <label
+                htmlFor={`epicSwitch${unique}`}
+                className={toggleStyles}
+                data-target="toggle"
+                tabIndex={0}
+            >
+                <div data-target="open" className={openButtonStyles}>
+                    Not a good time? Remind me later
+                </div>
+                <div data-target="close" className={closeButtonStyles} tabIndex={0}>
+                    <SvgClose />
+                </div>
+            </label>
```

Comment:
This control needs a bit more work to make it accessible to keyboard-only users and screen reader users. Rather than setting the checkbox to `display: none` you can make it invisible to sighted users:

```js
import {screenReaderOnly} from '@guardian/src-foundations/accessibility'


const checkboxStyles = css`
    ${screenReaderOnly};
    ...
`
```
```

Reply:
Sadly I can't make it keyboard-only and screen reader accessible without a bit of JS. If you try this out on the browser you'll see it works with keyboard-only navigation and screen readers just fine, but this was done with a couple of lines of JS, to trigger a click on the checkbox when user presses enter on the label - because the label also has two states (it's basically a toggle which shows and open button or close button depending on the `:checked` state of the checkbox. I don't think `visuallyHidden` will avoid the need for that.